apps/web/pages/event-types/[type].tsx

```diff
@@ -329,7 +329,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
     fetchTokens();
 
     !hashedUrl && setHashedUrl(generateHashedLink(eventType.users[0].id));
-  }, []);
+  }, [eventType.users, hashedUrl, tokensList]);
```

Comment:
This will refetch tokens infinitely. We need to stop and think what will happen with these changes instead of just applying the warning recommendations blindly.

Reply:
Again, sorry, I let my frustration with these warnings get the better part of me.